docs/faq/transformation-and-schemas.md

```diff
@@ -0,0 +1,21 @@
+# Transformation and Schemas
+
+### **Where's the T in Airbyte’s ETL tool?**
+
+Airbyte is actually an ELT tool, and you have the freedom to use it as an EL-only tool. The transformation part is done by default, but it is optional. You can choose to receive the data in raw \(JSON file for instance\) in your destination. 
+
+We do provide normalization \(if option is still on\) so that data analysts / scientists / any users of the data can use it without much effort. 
+
+We also intend to integrate deeply with DBT to make it easier for your team to continue relying you on them, if this was what you were doing. 
+
+### **How does Airbyte handle replication when a data source changes its schema?**
+
+How Airbyte handles data structure changes in a data source varies depending on the connector, as well as the replication method used for a given table within that connector.
+
+However, Airbyte decouples the Extract and Load of a source from the normalization of the data at the destination. So, the data will still land in your destination, but if you used any normalization, this part might be broken because of the schema change. 
```

Comment:
```suggestion
```

this isn't actually true. the best of my knowledge if we manage to write the raw we will successfully write the normalized one as well.